packager/media/formats/ttml/ttml_generator.cc

```diff
@@ -182,16 +213,18 @@ bool TtmlGenerator::AddSampleToXml(const TextSample& sample,
 bool TtmlGenerator::ConvertFragmentToXml(const TextFragment& body,
                                          xml::XmlNode* parent,
                                          xml::XmlNode* metadata,
+                                         std::set<std::string>& fragmentStyles,
```

Comment:
const reference if you are not modifying fragmentStyles.

Reply:
I actually do gather the styles so it should not const in this particular chain.